word_search.py

```diff
@@ -0,0 +1,102 @@
+"""
+This module contains word search solution
+"""
+import random
+import string
+from typing import Generator
+
+
+def get_valid_words(file_name: str) -> tuple:
+    """
+    Get all valid words from a file
+
+    :param file_name: Name of a file
+    :return: Tuple of valid words
+    """
+    with open(file_name) as words_dict:
+        words = words_dict.readlines()
+
+    return tuple([word.lower().strip() for word in words])
```

Comment:
what is the reason you chose a tuple as opposed to other data types?

Reply:
Because tuple is faster for parsing than list

Reply:
I think, we will save a few seconds, if we use tuple